packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
+  useConst,
+  useControllableState,
+  useId,
+  useIsSSR,
+  useMergedEventCallbacks,
+  useMergedRefs,
+  useMount,
+  useSSRContext,
+  useTimeout,
```

Comment:
```suggestion
```

Generally applications tend to restrict the usage of setTimeout unless there is a good reason to do so. Let's not encourage the use :)